pkg/security/podsecuritypolicy/provider.go

```diff
@@ -201,6 +201,10 @@ func (s *simpleProvider) ValidatePodSecurityContext(pod *api.Pod, fldPath *field
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("hostIPC"), pod.Spec.SecurityContext.HostIPC, "Host IPC is not allowed to be used"))
 	}
 
+	if !s.psp.Spec.HostUser && pod.Spec.SecurityContext.HostUser {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("hostUser"), pod.Spec.SecurityContext.HostUser, "Host User is not allowed to be used"))
```

Comment:
`Host User` sounds strange. `HostUser` or `Host user namespace` would be clearer. Otherwise, people wonder which user is meant.